pkg/logstash.sysv

```diff
@@ -96,12 +97,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      if [ $KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
+      if [ $LS_KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
```

Comment:
While we're here, can you quote the variable "$LS_KILL_ON_STOP_TIMEOUT" so it won't complain if someone has somehow unset this value?